api/src/pcapi/analytics/amplitude/events/booking_events.py

```diff
@@ -0,0 +1,20 @@
+from pcapi.analytics.amplitude.backends import amplitude_connector
+from pcapi.tasks import amplitude_tasks
+
+
+def track_book_offer_event(booking: bookings_models.Booking) -> None:
+    amplitude_tasks.track_event.delay(
+        amplitude_tasks.TrackAmplitudeEventRequest(
+            user_id=booking.userId or 0,
```

Comment:
Il me semble que `Booking.userId` n'est plus jamais nul, en réalité. Il l'était, je crois, quand on avait des `EducationalBooking`. Mais ils ont tous été migrés vers des `CollectiveBooking`.

Je propose d'ajouter un FIXME ici (pour supprimer ce `or 0`) et de créer une pull request pour l'ajout d'une contrainte NOT NULL sur ce champ.